app/src/main/java/ca/marshallasch/veil/tagList/TagListAdapter.java

```diff
@@ -0,0 +1,179 @@
+package ca.marshallasch.veil.tagList;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.CheckBox;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import ca.marshallasch.veil.R;
+
+/**
+ * This class is the the adapter for the list of tags
+ *
+ * This was built based off of:
+ * https://stackoverflow.com/questions/38417984/android-spinner-dropdown-checkbox/38418249#38418249
+ * @author Marshall Asch
+ * @version 1.0
+ * @since 2018-07-24
+ */
+public class TagListAdapter extends ArrayAdapter<ListState>
+{
+    private Context context;
+    private ArrayList<ListState> tagList;
+    private boolean isFromView = false;
+
+    /**
+     * This class will hold the view view for the cell.
+     */
+    private class ViewHolder {
+        private TextView textView;
+        private CheckBox checkBox;
+    }
+
+    /**
+     * Constructor with the default list of tags.
+     * @param context the application context
```

Comment:
change "application" in `@param context the application context` to "activity's."